src/styles/pages/_donate.scss

```diff
@@ -173,6 +173,16 @@ body.donate-page {
       text-align: center;
       background: $white;
     }
+
+    .ask-text-small {
+      font-size: 0.5em;
+      font-weight: 400;
+
+      a {
+        color: $black;
```

Comment:
Why are links here black? Will that cause confusion with ask-sub-text?

Reply:
The links are black because the new design has a blue background there, that is exactly the same color as our default link text color. You can see an example here: https://act.moveon.org/donate/vote-sticker-thanks-september/

Reply:
Thank you! That makes it much clearer.